public/main.css

```diff
@@ -10,21 +10,90 @@ html {
 }
 
 body {
-  min-height: 100%;
-  position: relative;
+  display: flex;
+  flex-direction: column;
   margin: 0;
+  min-height: 100vh;
+  position: relative;
+}
+
+.header {
+  align-items: flex-start;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding: 31px;
+  width: 100%;
 }
 
 .send-logo {
+  display: flex;
   position: relative;
-  top: 31px;
-  left: 31px;
-  display: inline-block;
+  align-items: center;
+
+}
+
+.site-title {
+  font-size: 34px;
+  font-weight: 500;
+  margin: 0;
+  position: relative;
+  top: -1px;
+}
+
+.site-subtitle {
+  font-size: 12px;
+  margin: 0 8px;
+  color: #0C0C0D;
+}
+
+.site-subtitle a {
+  font-weight: bold;
+  color: #0C0C0D;
+  transition: color 50ms;
+}
+
+.send-logo:hover a {
+  color: #0297F8;
+}
+
+.feedback {
+  background-color: #0297F8;
+  background-image: url('resources/feedback.svg');
+  background-position: 4px 6px;
+  background-repeat: no-repeat;
+  background-size: 14px;
+  border-radius: 3px;
+  border: 1px solid  #0297F8;
+  box-shadow: 0 1px 2px rgba(0,0,0,.1);
```

Comment:
I think this will fail our CSS linting rules too. Probably need spaces after commas and leading zeros on the numbers. Sorry.  `$ npm run format` and `$ npm run lint` may be able to catch/fix some of these.